### PR TITLE
feat(TPSVC-1087/fonts): add fonts to local repo

### PR DESCRIPTION
**Pull request by slavasel**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Fonts are loaded by npm. Module belongs to some github user and it's different to our fonts in frontity and Goggle fonts.

**What is the new behavior?**
Fonts are stored in our repo. We include them in _fonts.scss. The main issue is that relative paths to fonts will remain still relative after including this repo to another projects. 
But this is well known and discussed issue. imo, the best way to solve it on client side is to use resolve_url loader in webpack. 

**Does this PR introduce a breaking change?**

- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...
For including this repo to other projects, user will need to use resolve-url loader for resolving relative paths.

**Other information**:

### CODE REVIEW

**Comment by mtikhinskaya** [comment]
Please add link to your project that compares fonts from different sources

**Comment by slavasel** [comment]
https://github.com/slavasel/fonts_demo

**Comment by jmfrancois** [comment]
https://jmfrancois.github.io/fonts_demo/

**Comment by jmfrancois** [comment]
@slavasel could you please fill the form ?
What is the current behavior
What is the new behavior ?

**Comment by slavasel** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ Edit your Sass entry file.
 @import '~bootstrap-talend-theme/src/theme/bootstrap';
 ```
 
+To add fonts to your project you will need to add resolve-url loader:
+(https://github.com/bholloway/resolve-url-loader), because fonts, images
+in this repo with relative paths will be computed by sass as local ones.
+Example of usage:
+{
+    test: /\.scss$/,
+    loader: `style!css!resolve-url!sass?sourceMap`,
```
Comment (slavasel) [inline README.md]: Without resolve-url loader in TPSVC, fonts can not be found. Webpack is trying to find them in ./fonts inside our project. So I believe it's necessary.

Comment (mtikhinskaya) [inline README.md]: But now fonts is also in different folder, right? So you will use resolve-url in any case, won't you?

Comment (slavasel) [inline README.md]: yes, resolve-url will be used in any case in projects, using bootstrap-theme

Comment (mtikhinskaya) [inline README.md]: Then it's out of scope this pr. Please separate this changes

Comment (slavasel) [inline README.md]: This change (about resolve-url) is only an information in README.md. Everything else is in the scope (as I understand), because I removed dependency and added offline fonts. Do you want me to remove documentation changes?

Comment (mtikhinskaya) [inline README.md]: Yes, I think it's better to move this doc changes to another PR. Especially when they are not caused by font changes

Comment (slavasel) [inline README.md]: Ok, I see. Will do it. Thanks!
